package.json

```diff
@@ -67,6 +66,8 @@
 	"dependencies": {
 		"@googlemaps/markerclusterer": "^2.4.0",
 		"google-map-react": "^2.2.1",
-		"locutus": "^2.0.16"
+		"locutus": "^2.0.16",
+		"lodash.debounce": "^4.0.8",
+		"lodash.throttle": "^4.1.1"
```

Comment:
Does this cause it to bundle that in the `front.js` build, rather than using the version that ships w/ Core?

If so, I wonder if that's actually worse for holistic performance, since the Core script would already be cached locally in a lot of places. I'm guessing that wouldn't show up in Lighthouse scores b/c it's not realistic enough to detect it.

Reply:
Yeah it does cause it to be bundled. Sorry, I wasn't aware of the core dependency, and it wasn't clear to me looking at our imports, where the comments list it as an External Dependency, and not a Wordpress one:

![Screenshot 2023-12-11 at 8 36 22 AM](https://github.com/WordPress/wporg-mu-plugins/assets/1017872/0ec0b6c6-9d06-4e75-8d2c-ca59d0cc8801)